packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -186,6 +190,21 @@ class Transforms {
           activity.participantDisplayNames = space.participantDisplayNames.filter((value) => value !== activity.actorDisplayName).toString();
         }
 
+        // only post and share activities have content which needs to be decrypted
+        if (activity.verb !== 'post' && activity.verb !== 'share') {
```

Comment:
['post', 'share'].includes(acitivit.verb)

may be more flexible for the future. I imagine we'll be adding an 'edit' verb (assuming it's not covered by post).